doc/site/articles/select-command.markdown

```diff
@@ -58,11 +58,11 @@ Here are the filters. Note that "units" generally means both buildings and mobil
 
 ### `Builder`
 
-  Keep only units and buildings that can construct. This means Factories, Construction Turrets, Constructors, but not Rezzers.
+  Keep only units and buildings that can do any sort of worker tasks. This means Factories, Construction Turrets, Constructors, Rezzers and whatnot.
```

Comment:
Why did this change? Do you know?

Reply:
Did it? The filter [hasn't changed in 6 years](https://github.com/beyond-all-reason/spring/blame/master/rts/Game/UI/SelectionKeyHandler.cpp#L120) and the condition checked by filter [hasn't changed in 11 years](https://github.com/beyond-all-reason/spring/blame/master/rts/Sim/Units/UnitDef.h#L69). Neither of them seem to make an exception for rezzing.

So I guess the doc has just always been incorrect. You wrote most of the docs so I sort of lazily assumed by default you were the one to write it (which is why I marked you specifically for review) but now I checked and it looks like it was added in #239 by @ecraven. If he shows up and says that he's certain that this is how it worked back then (since the rezzer exception looks pretty purposeful) I guess we could investigate further.

Reply:
Hmmm... I see, maybe my memory is not good on this matter then because I remember it used to work as it advertises